packages/web-media-box/src/pipelines/mse/consts/sourceBuffer.ts

```diff
@@ -0,0 +1,5 @@
+export const OperationType = {
+  APPEND: 'append',
+  ABORT: 'abort',
```

Comment:
Do we need `abort` for source buffers? I really think if we already downloaded and processed segment - just append it to the source buffer

Reply:
Agreed for `append` however do we want to be able to call `abort` in the case where we're seeking or switching content types (ABR or alternate audio tracks) and want to write the new content to the buffer ASAP?

Reply:
I suspect this will be like a very tiny delay since the only window we can abort is when we already called appendData but haven't received updateend event...

We can keep abort... at this point, I don't know if it makes any sense since there might be a chance it is fast to keep things going instead of resetting everything...

Reply:
Yeah that's a very good point. I haven't done any testing recently, I vaguely remember there being some pretty slow `appendData` -> `updateend` events fired on some old smart TVs years ago, but it may be far less of an issue now. I'm cool with just omitting abort for now, especially if get rid of `OperationType` due to it not being used currently.